_posts/2023-08-13-kafka-node-pools-introduction.md

```diff
@@ -0,0 +1,347 @@
+* We might be still evaluating it this is a feature we want to have, whether it fulfills its goals, and provides good user experience.
+* It might still change in future releases or even be removed.
+  It might not maintain a backwards compatibility with the previous versions of the feature gate.
+
+All of the above means that you should probably not use Node Pools in your production environment just yet.
+But at the same time, it offers a preview of how Strimzi might look like in few months and shows some new features that will be possible.
+So we encourage you to try it in some test environment and share your feedback, bugs or improvement ideas with us.
+
+The current plan is that if everything goes well, the `KafkaNodePools` feature gate will move to the _beta_ phase and be enabled by default in Strimzi 0.39.
+But the plan might of course change depending on how well it works and on the feedback we receive.
+
+If you want to learn more about feature gates in Strimzi, you can learn more about them in [our documentation](https://strimzi.io/docs/operators/0.36.1/full/deploying.html#ref-operator-cluster-feature-gate-releases-str).
+You can also find there a full list of the [feature gates Strimzi currently supports](https://strimzi.io/docs/operators/0.36.1/full/deploying.html#ref-operator-cluster-feature-gates-str).
+
+### Enabling Node Pools
+
+To enable the Node Pools, you have to enable the `KafkaNodePool` feature gate.
+You can do that by editing the Strimzi Cluster Operator Deployment and adding `+KafkaNodePools` to the `STRIMZI_FEATURE_GATES` environment variable:
+
+```yaml
+env:
+  # ...
+  - name: STRIMZI_FEATURE_GATES
+    value: +KafkaNodePools
+```
+
+You can find more details about enabling the feature gates in [our documentation](https://strimzi.io/docs/operators/0.36.1/full/deploying.html#ref-operator-cluster-feature-gates-str).
+
+Enabling the `+KafkaNodePools` does not have any impact on your existing Kafka clusters.
+The Kafka Node Pools will be used only for the Kafka clusters with the annotation `strimzi.io/node-pools: enabled` on the `Kafka` custom resource.
+Once the feature gate is enabled and the `Kafka` resource has this annotation, the operator will look for the `KafkaNodePool` custom resources will use Node Pools for this Kafka cluster.
+For such resource, no Kafka Pods will be created based on the `Kafka` resource itself but only based on the `KafkaNodePool` resources.
+
+### Configuring Node Pools
+
+The Node Pools are configured using a new custom resource named `KafkaNodePool`.
+It currently supports 6 different configuration options:
+* Number of replicas
+* Role(s) of the nodes in this pool
+* The storage configuration
+* Resource requirements
+* JVM configuration options
+* Template for customizing the resources belonging to this pool such for example Pods or containers
+
+These are the options which can be different between various node pools.
+Each node pool can configure them differently.
```

Comment:
Aren't the two sentences above saying the same thing? Maybe we can just remove one of them?